cpp/include/cudf/transform.hpp

```diff
@@ -111,5 +111,30 @@ std::pair<std::unique_ptr<cudf::column>, std::unique_ptr<cudf::column>> encode(
   cudf::column_view const& input,
   rmm::mr::device_memory_resource* mr = rmm::mr::get_default_resource());
 
+/**
+ * @brief Creates a boolean column from given bitmask.
+ *
+ * Each bit represents one value, 0 - false and 1 - true.
+ * It is user's responsibility to make sure the bitmask agrees
+ * with given offset and length.
+ *
+ * Examples:
+ * @code{.pseudo}
+ * input: {147}
+ * output: [{false, true, false, true, false, true, false, true}]
+ * @endcode
+ *
+ * @param bitmask A device pointer to the bitmask which needs to be converted
+ * @param offset position of the bit from which the conversion should start
+ * @param length Number of bits needs to be converted from offset position
+ * @param mr Device memory resource used to allocate the returned columns's device memory
+ * @return A boolean column representing the given mask from [offset, offset+length).
+ */
+std::unique_ptr<column> mask_to_bools(
+  bitmask_type const* bitmask,
+  size_type offset,
```

Comment:
For consistency with other mask related APIs, I'd prefer to see this take a `begin_bit` and `end_bit`.